portal/src/graphql/portal/VerificationConfigurationScreen.tsx

```diff
@@ -96,9 +98,21 @@ function constructConfig(
     if (initialState.email.required !== currentState.email.required) {
       v.claims.email.required = currentState.email.required;
     }
+    if (currentState.email.required === true) {
+      currentState.email.enabled = true;
+    }
```

Comment:
This logic should be applied to the event handler so that the developer can see the relationship.